lib/facets/aggs-all.js

```diff
@@ -81,43 +81,33 @@ module.exports = function (queryParams) {
 function filter (queryParams, exclude) {
   var filters = [];
   // category
-  if (exclude !== 'category') {
-    const categories = queryParams.filter.objects.categories;
-    if (categories) {
-      filters.push({ terms: { 'categories.value': categories } });
-    }
+  const categories = queryParams.filter.objects.categories;
```

Comment:
The function [queryParams](https://github.com/TheScienceMuseum/collectionsonline/blob/60fffb74b957ee2b0e1bc5b1f4a3e069f9478b7e/lib/query-params.js#L32-L33) define the parameters object based on the query parameters of the requests. The function define a full object where all the necessary properties should be defined (the leaf/last properties can be null). So I think queryParams.filter.objects will always be [defined](https://github.com/TheScienceMuseum/collectionsonline/blob/60fffb74b957ee2b0e1bc5b1f4a3e069f9478b7e/lib/query-params.js#L16)